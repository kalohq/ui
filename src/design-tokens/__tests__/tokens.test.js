/* @flow */
/* eslint-env jest */
import renderer from 'react-test-renderer';

import JSTokens from '../tokens.js';
import CSSTokens from '../tokens.css.js';
import ThemeTokens from '../tokens.theme.js';

describe('Tokens', () => {
  test('should be present', () => {
    expect(JSTokens).toMatchSnapshot();
  });

  test('should exist in CSS form', () => {
    expect(CSSTokens).toBeDefined();
  });

  test('should exist in theme form', () => {
    expect(ThemeTokens).toBeDefined();
  });
});
