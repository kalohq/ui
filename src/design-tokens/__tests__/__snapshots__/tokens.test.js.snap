// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tokens should be present 1`] = `
Object {
  "black": "rgb(0, 0, 0)",
  "blue300": "rgb(193, 244, 255)",
  "blue400": "rgb(101, 210, 234)",
  "blue500": "rgb(84, 195, 219)",
  "blue600": "rgb(77, 184, 207)",
  "blue700": "rgb(43, 167, 194)",
  "borderRadius": "4px",
  "borderRadiusButton": "4px",
  "borderRadiusInput": "4px",
  "fontSizeHeadingLarge": "32px",
  "fontSizeHeadingMedium": "24px",
  "fontSizeHeadingSmall": "16px",
  "fontSizeHeadingXSmall": "12px",
  "fontSizeTextLarge": "18px",
  "fontSizeTextMedium": "16px",
  "fontSizeTextSmall": "14px",
  "fontSizeTextXSmall": "12px",
  "fontWeightLight": "300",
  "fontWeightMedium": "500",
  "fontWeightNormal": "400",
  "fontWeightSemibold": "600",
  "green300": "rgb(216, 240, 232)",
  "green400": "rgb(79, 210, 164)",
  "green500": "rgb(62, 179, 138)",
  "green600": "rgb(56, 160, 124)",
  "green700": "rgb(41, 134, 101)",
  "grey200": "rgb(249, 250, 252)",
  "grey300": "rgb(238, 244, 250)",
  "grey400": "rgb(208, 216, 223)",
  "grey500": "rgb(176, 186, 195)",
  "grey600": "rgb(137, 146, 155)",
  "grey700": "rgb(112, 125, 138)",
  "navy300": "rgb(249, 250, 252)",
  "navy400": "rgb(198, 199, 216)",
  "navy500": "rgb(147, 166, 193)",
  "navy600": "rgb(84, 103, 137)",
  "navy700": "rgb(55, 69, 97)",
  "orange300": "rgb(255, 237, 180)",
  "orange400": "rgb(255, 206, 94)",
  "orange500": "rgb(253, 184, 28)",
  "orange600": "rgb(246, 175, 13)",
  "orange700": "rgb(216, 145, 24)",
  "pink300": "rgb(251, 223, 226)",
  "pink400": "rgb(253, 179, 200)",
  "pink500": "rgb(234, 95, 110)",
  "pink600": "rgb(224, 88, 103)",
  "pink700": "rgb(195, 74, 87)",
  "purple300": "rgb(232, 226, 242)",
  "purple400": "rgb(159, 129, 218)",
  "purple500": "rgb(138, 112, 189)",
  "purple600": "rgb(119, 95, 168)",
  "purple700": "rgb(96, 78, 131)",
  "white": "rgb(255, 255, 255)",
}
`;
